packages/react-static/src/static/__mocks__/defaultConfigProduction.mock.js

```diff
@@ -19,6 +18,11 @@ export default {
     STATIC_DATA: './root/dist/staticData',
   },
   terser: {
-    terserOptions: {},
+    terserOptions: {
+      compress: {},
```

Comment:
Can you explain what this fixed/changed? I just don't know 😅

Reply:
I hoped you would know :P From what I can tell, the default config includes these nested empty objects, while the obsolete tests were still expecting just `terserOptions: {}`

Reply:
Hm, I think I also solved this with my other fix. Let me check

Reply:
Indeed! Let me cleanup my commits a bit and we should be good to go.